docs/i18n/overview.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: i18n Overview
+name: Overview
+parent: i18n
+nav_order: 2
+---
+
+# Overview i18n Overview
+Many companies use SAP systems and applications and many different countries.
+To ensure easy translation, the "i18n" (Internationalization) process is used.
+The app is not always recreated, but only the texts are translated and replaced.
+
+## Language Tags
+To identify the individual languages, so-called "Language Tags" are used which are defined by [BCP-47] (http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
```

Comment:
BCP-47 should be the link text, right? so removing the " " between [...] and (...) fixes that

Reply:
changed